sdk/common/single_async_operation.go

```diff
@@ -0,0 +1,49 @@
+package common
```

Comment:
Hmm why is this in the SDK? We generally want only the code we "export' here. I.e. if I implement a solution on top of NSM I will import the SDK/* Or maybe this iwll be used int he follow up PRs?

Reply:
thanks, fixed.